test/ruby_lsp_rails/hover_test.rb

```diff
@@ -44,17 +44,15 @@ class User < ApplicationRecord
           User
         RUBY
 
-        assert_equal(<<~CONTENT, response.contents.value)
+        assert_equal(<<~CONTENT.chomp, response.contents.value)
```

Comment:
(nit) Why the change to using `chomp`? (I would have thought it better if the test shows the actual whitespace.)

Reply:
It's the heredoc literal that contains the extra line break. The actual result doesn't include it, which is a result of using `join` on the entries.

So showing the line break would require something like
```ruby
assert_equal(<<~CONTENT, "#{response.contents.value}\n")
```